src/main/java/com/sequenceiq/cloudbreak/service/cluster/AmbariClusterService.java

```diff
@@ -195,11 +194,10 @@ public ClusterStatusUpdateRequest updateStatus(Long stackId, StatusRequest statu
                 flowManager.triggerClusterStart(retVal);
             }
         } else {
-            if (!Status.AVAILABLE.equals(clusterStatus)) {
+            if (clusterNotReadyForStop(clusterStatus)) {
```

Comment:
Why should we check that the cluster is in 'STOPPED' state?
If it is stopped we should not try to stop it again.